18F/identity-idp
# LG-13355: filter piv upsell

## 🎫 Ticket

Link to the relevant ticket:
[LG-13355](https://cm-jira.usa.gov/browse/LG-13355)

## 🛠 Summary of changes
This leverages using the Fed domain list to ensure we are only prompting users with Fed emails to add PIV or .mil users. 

## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

- Locally you can either download the full fed domain path by running `ruby lib/fed_email_domain_downloader.rb ` to download the file locally. Move that file from tmp folder to fed_domain_emails folder. Or use the sample fed_email_domain sample file in the folder. 
- Set fed_domain_file_path to the `fed_email_domain.txt` and set use_fed_domain_file to `true`
- Create an account with .mil or a valid .gov email domain,
- Be directed to add a piv for valid domains or .mil. Ensure that when using an invalid domain you are not prompted for Emails.
